src/components/SearchBar/SearchBar.module.scss

```diff
@@ -6,32 +6,54 @@
 .searchBar input {
 	width: 100%;
 	height: 32px;
-	background: var(--color-cosmic-gradient-25);
+	background: none;
+	border: 1px solid var(--color-purple-lighter);
 	border-radius: 31px;
 	padding: 0 24px 1px 39px;
 
 	/* Text */
 	text-align: left;
-	color: white;
+	color: #bfbfbf;
```

Comment:
Change this to `var(--color-grey-lighter-2)`

Reply:
Done. Thank you.

Reply:
Done.